ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -382,13 +382,22 @@ - (NSString *) getIpAddress {
         temp_addr = interfaces;
         while(temp_addr != NULL) {
             if(temp_addr->ifa_addr->sa_family == AF_INET) {
-                // Check if interface is en0 which is the wifi connection on the iPhone
-                if([[NSString stringWithUTF8String:temp_addr->ifa_name] isEqualToString:@"en0"]) {
-                    // Get NSString from C String
-                    address = [NSString stringWithUTF8String:inet_ntoa(((struct sockaddr_in *)temp_addr->ifa_addr)->sin_addr)];
-
-                }
-
+                NSString* ifname = [NSString stringWithUTF8String:temp_addr->ifa_name];
+                #if TARGET_OS_TV
+                    if(
+                        // Check if interface is en0 which is the ethernet connection on the Apple TV
+                        [ifname isEqualToString:@"en0"] ||
+                        // Check if interface is en1 which is the wifi connection on the Apple TV
+                        [ifname isEqualToString:@"en1"]
```

Comment:
out of curiosity, why even make this '#if TARGET_OS_TV', is there any harm en just adding the en1 check to the main case and making it the only case?

Reply:
I don't know if it might affect in any way the logic for iPhone. That's why I put it in a separate block. I can update the PR though, let me know!

Reply:
I have an iPhone and can test it, it seems like it would Just Work, and the simpler the better

Reply:
by which I mean, please update the PR, I'll pull it and check :-)

Reply:
Thanks, the PR is updated now